build_tools/setup_helpers/extension.py

```diff
@@ -39,11 +39,15 @@ def _get_build(var, default=False):
 _BUILD_RNNT = _get_build("BUILD_RNNT", True)
 _USE_ROCM = _get_build("USE_ROCM", torch.cuda.is_available() and torch.version.hip is not None)
 _USE_CUDA = _get_build("USE_CUDA", torch.cuda.is_available() and torch.version.hip is None)
+_USE_OPENMP = _get_build("USE_OPENMP", True) and 'ATen parallel backend: OpenMP' in torch.__config__.parallel_info()
```

Comment:
TODO: double check this the later condition is necessary. It might work without it in macOS. (but not sure if that's okay. Consult someone from PyTorch core)